lib/themes/dosomething/paraneue_dosomething/js/app.js

```diff
@@ -50,6 +51,8 @@ $(document).ready(function() {
   // Initialize the donation form on the donate page.
   Donate.init();
 
+  Globe.init();
```

Comment:
I think it'd be better to run this at the very beginning of the `$(document).ready()`, so it's better change for the request to pull through if user hits refresh or leaves the page.